components/dash-core-components/tests/integration/markdown/test_markdown.py

```diff
@@ -93,3 +93,56 @@ def test_mkdw002_dcclink(dash_dcc):
     dash_dcc.percy_snapshot("mkdw002 - markdowns display")
 
     assert dash_dcc.get_logs() == []
+
+
+def test_mkdw003_without_mathjax(dash_dcc):
+    app = Dash(__name__, eager_loading=False, assets_folder="../../assets")
+
+    app.layout = html.Div(
+        [
+            dcc.Markdown("# No MathJax: Apple: $2, Orange: $3"),
+        ]
+    )
+
+    dash_dcc.start_server(app)
+    dash_dcc.percy_snapshot("mkdw003 - markdown without mathjax")
```

Comment:
```suggestion
    dash_dcc.wait_for_text_to_equal("h1", "No MathJax: Apple: $2, Orange: $3")
    assert not dash_dcc.driver_execute_script("return !!window.MathJax")
```

We aren't testing anything particularly visual here - so I'd rather we not add a snapshot, instead just make DOM & JS assertions